src/main.c

```diff
@@ -65,16 +67,16 @@ int main() {
     SPI_ProtoInit();
     CONFIG_ReadDisplay();
     CONFIG_ReadModel(CONFIG_GetCurrentModel());
-    CONFIG_ReadLang(Transmitter.language);
+    CONFIG_ReadLang(Transmitter.language); //diff15
 
-    BACKLIGHT_Brightness(Transmitter.backlight);
+    BACKLIGHT_Brightness(Transmitter.backlight); //diff16
     LCD_Contrast(Transmitter.contrast);
-    LCD_SetFont(DEFAULT_FONT.font);
+    LCD_SetFont(DEFAULT_FONT.font); //diff17
```

Comment:
Lint Error:
main.c:70:  At least two spaces is best between code and comments  [whitespace/comments] [2]
main.c:70:  Should have a space between // and comment  [whitespace/comments] [4]
main.c:72:  At least two spaces is best between code and comments  [whitespace/comments] [2]
main.c:72:  Should have a space between // and comment  [whitespace/comments] [4]
main.c:74:  At least two spaces is best between code and comments  [whitespace/comments] [2]
main.c:74:  Should have a space between // and comment  [whitespace/comments] [4]